### DISCUSSION TITLE
bad request: role must be "user", got "assistant"

### DISCUSSION THREAD

**Discussion by CODEFORYOU69** [community/community#56119] [Copilot Conversations]
### Select Topic Area

Question

### Body

Hello 

sorry for my english

i installed vscode insiders for use the new github copilot chat and i have an issues with this message 

unhandled status from server: 400 Bad request: bad request: role must be "user", got "assistant"

Do you know how to fix it ?

**Comment by TheHumanistX** [comment]
I also got this AND got unhandled status from server: 400 Bad request: bad request: role must be "user", got "server"

All I was doing was testing a longer prompt...? No idea what to do with this.

**Comment by TheTwilightStorm** [comment]
Bumping this, I also keep getting this error. I also tried to send a longer prompt. Seems like long prompts are causing errors for Copilot Chat.

**Comment by Anaximand** [comment]
Same here
